arangodantic/tests/test_model.py

```diff
@@ -425,3 +425,24 @@ async def test_find_one_with_sort(identity_collection):
 
     found = await Identity.find_one(sort=[("name", DESCENDING)])
     assert found.name == "Bob"
+
+
+@pytest.mark.asyncio
```

Comment:
These tests don't seem to test that the indexes are created, it only tests they are correct IF they are created, right?

Reply:
What I mean is that the logic was

- get list of indexes
- if specific index found
- check that index is as expected

it did not check for

1) extra indexes of different types
2) missing indexes